### ISSUE TITLE
Flickr

### ISSUE THREAD

**Issue by tonespy** [SimplicityMobile/Simplicity#2]
Can you create a provider for Flickr?

**Comment by edjiang** [comment]
Yeah, definitely! Btw, how'd you find this repo? Haven't quite published v1 just yet, but planning on getting it out tomorrow morning! =]

Looks like Flickr uses OAuth 1.0a, so I would need to build the server-side component in #1 to make it work fully securely =]

**Comment by tonespy** [comment]
I was sniffing through Github for a solution to a flickr implementation I was doing using Sailsjs/Nodejs and then I saw you module. On reading through, it was what I was working on. Since you're working on it already, I wouldn't mind if you finish for me to use 👍 💯

**Comment by edjiang** [comment]
Awesome. I might not get to doing the companion server for maybe ~2-3 weeks, just as a FYI.

**Comment by tonespy** [comment]
Hey. Just checking in again

**Comment by edjiang** [comment]
Hey! Probably going to get started on the companion server at the end of the next week; I'm doing two talks next week, so need to prepare =]
